examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -242,6 +242,7 @@ CHIP_ERROR PairingCommand::PairWithCode(NodeId remoteId)
     }
 
     ReturnErrorOnFailure(MaybeDisplayTermsAndConditions(commissioningParams));
+    CurrentCommissioner().RegisterPairingDelegate(this);
```

Comment:
This also seems unrelated with "move signal handling out of SDK".

Why these changes? Can it be a separate PR? Or explain the changes in the summary.

Reply:
It's a problem I found while moving signal handler out that PairingCommand::OnPairingComplete() won't be called if stopping it unexpectedly.